packages/livekit-rtc/src/transcription.ts

```diff
@@ -0,0 +1,14 @@
+export interface TranscriptionSegment {
+  id: string;
+  text: string;
+  startTime: number;
+  endTime: number;
```

Comment:
should these be BigInts here to begin with?

Reply:
yeah that makes sense. of course they're all 0 everywhere anyways!